pkg/kubectl/cmd/util/helpers.go

```diff
@@ -204,25 +206,28 @@ func StandardErrorMessage(err error) (string, bool) {
 }
 
 // MultilineError returns a string representing an error that splits sub errors into their own
-// lines. The returned string will end with a newline.
+// lines. The returned string will end with a newline. The prefix should not end in . or : or \n.
 func MultilineError(prefix string, err error) string {
+	if err == nil {
+		return fmt.Sprintf("%s.\n", prefix)
```

Comment:
In fact it's pretty strange to call MultilineError with a nil error. I will check whether this can be cleaned up easily.